test/pulpcore_plugin_integration_test.rb

```diff
@@ -15,7 +15,12 @@ def app
   end
 
   def test_features
-    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulpcore.yml').returns(enabled: true, pulp_url: 'http://pulpcore.example.com/foo', content_app_url: 'http://pulpcore.example.com:24816/')
+    Proxy::DefaultModuleLoader.any_instance.expects(:load_configuration_file).with('pulpcore.yml').returns(
+      enabled: true,
+      pulp_url: 'http://pulpcore.example.com/pulpcore',
+      content_app_url: 'http://pulpcore.example.com:24816/',
+      rhsm_url: 'https://smartproxy.example.com/rhsm'
```

Comment:
Perhaps this should keep the pattern and show it can be multi homed?
```suggestion
      rhsm_url: 'https://rhsm.example.com/rhsm'
```

Reply:
I thought about naming it `candlepin.example.com/rhsm` but of course it doesn't have to actually point to the candlepin server, just an apache reverse proxy to it.

in some sense, smartproxy.example.com vs. pulpcore.example.com already demonstrates the multi-homing support, right? but i'll go with rhsm.example.com to keep with a consistent and easy to follow pattern

Reply:
done.

I thought about candlepin.example.com but talked myself out of it because it doesn't have to be an actual candlepin server, just a webserver that redirects to it